addons/l10n_it_edi/models/account_move.py

```diff
@@ -2047,3 +2305,56 @@ def _l10n_it_edi_get_message(self, transformed_notification):
                 file=filename, partner=partner_name))
         }
         return new_state_messages_map.get(new_state)
+
+    # -------------------------------------------------------------------------
+    # Import
+    # -------------------------------------------------------------------------
+
+    @api.model
+    def _get_tag(self, element, selector):
+        if element is None:
+            return None
+        sub = element.xpath(selector)
+        if sub is None or len(sub) == 0:
+            return None
+        return sub[0]
+
```

Comment:
```suggestion
```
We can finally remove this and use the `get_text/float...` functions at the beginning of the file.